bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/EspressoHouse.java

```diff
@@ -51,7 +52,7 @@ public EspressoHouse(Context context) {
         NAME = "Espresso House";
         NAME_SHORT = "espressohouse";
         BANKTYPE_ID = IBankTypes.ESPRESSOHOUSE;
-        INPUT_TYPE_USERNAME = InputType.TYPE_CLASS_TEXT | + InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS;
+        INPUT_TYPE_USERNAME = InputType.TYPE_CLASS_TEXT | +InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS;
```

Comment:
It's actually based on _Android Studio's_ code formatting so I thought it would be good..